### DISCUSSION TITLE
Targets, Quarto, and Python

### DISCUSSION THREAD

**Discussion by nviets** [ropensci/targets#901] [Ideas]
It's exciting to see Targets has already begun supporting Quarto documents. Do you expect [python chunks](https://quarto.org/docs/computations/python.html) within a quarto document to work inside a targets plan? That would be really neat, since targets is head and shoulders above the workflow management tools native to python. I imagined something like:

```
```{targets-python some-globals, tar_globals = TRUE, tar_interactive = TRUE}
import keras
x = 1
if x == 1:
    print("x is 1.")
```

Paired with arrow, data might even be passed agnostically between mixed language chunks.

**Comment by wlandau** [comment]
Thanks @nviets. Python chunks themselves should work, as well as all the usual Quarto magic, but they won't be scanned for upstream `tar_read()` target dependencies.
